orchagent/routeorch.cpp

```diff
@@ -1282,6 +1282,15 @@ bool RouteOrch::addRoute(RouteBulkContext& ctx, const NextHopGroupKey &nextHops)
 
         if (nexthop.ip_address.isZero())
         {
+            if(gPortsOrch->isInbandPort(nexthop.alias))
```

Comment:
This is applied for both inband port and vlan, right?

Reply:
Yes. since static routes of system neighbors are added for both types of inband, this is applicable for both.